docs/source/getting-started/install.rst

```diff
@@ -100,6 +100,12 @@ ScanCode needs a Python 3.6 (*highly recommended*) or a Python 2.7 interpreter.
 
       64-bit Python interpreters (x86-64) are the only interpreters supported by
       Scancode on all operating systems which means only 64-bit Windows is supported.
+      
+      If you have more than one installed Python version(eg: Python3.9, Python2.7).
+      You can check all Python version installed in your local machine by running the below code::
+      
+         C:\Users>py -0
```

Comment:
Are you sure that this will always be a  `C:\Users>` prompt?

Reply:
@pombredanne I tried running this command as C:\Users\Sadhana>py -0 it give the same result. Please check this screenshot
![image](https://user-images.githubusercontent.com/56973448/112711612-f5785d00-8e86-11eb-8908-4ff76753eab9.png)

Reply:
And by running this command _C:\Users>py -0p_ it gives the version+path
![image](https://user-images.githubusercontent.com/56973448/112711780-3d4bb400-8e88-11eb-878a-7224bd1c3874.png)

Reply:
The area where you are changing is the general section in docs and if you are changing that for Windows I think you should also include for other os.
And I think C:\Users as we can get the py versions from other directories too :)

Reply:
Hi @Pratikrocks , sorry for the late response, if I am not getting wrong so you are saying that I should also add commands to know multiple python version in other OS like Mac and Linux, right?

Reply:
Yes, since it's a general section.
And moreover I think prefixing the directory for checking py versions in windows is not appropriate as we can access it from ay directories too :)

Reply:
@Pratikrocks , so I C:\Users\User_Name>py -0 will be correct?

Reply:
Hmm not really , as we can run your command from d drive / f. Drive too right?

Reply:
@Pratikrocks , I think the command is only supported where your python file is store. As in my case I have stored python in C drive.

Reply:
Oh okay cool 👍